Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml

```diff
@@ -1148,11 +1148,26 @@
         </Target>
     </ECRelationshipClass>
 
-    <ECEntityClass typeName="LoadCombination" modifier="Sealed" displayLabel="Load Combination" description="Combination of LoadConditions.">
+    <ECEntityClass typeName="LoadCombination" modifier="Sealed" displayLabel="Load Combination (Deprecated)" description="DEPRECATED">
```

Comment:
@diegoalexdiaz, Laurynas mentioned that there is a BIS validation rule that prohibits duplicated `displayLabel`s in the Schema. Perhaps this rule should not be applied to deprecated classes?